src/wp-includes/css/custom-properties.css

```diff
@@ -92,6 +92,16 @@ body {
 
 	--wp-admin--plugin-modal-banner-title--color: #fff;
 	--wp-admin--plugin-modal-banner-title--background: rgba(29, 35, 39, 0.9);
+
+	--wp-admin--login-anchor--color--hover: #135e96;
+	--wp-admin--login-anchor--color--focus: #043959;
```

Comment:
These could be `--wp-admin--theme--primary` & `--wp-admin--theme--primary--step-10` - right now step-10 is not actually in the codepen list of colors, but it could be updated to use blue-70 (#0a4b78). That can happen later when we're ready to audit the new properties though. 🤔

Reply:
@ryelle Would you like me to swap these? I know they're close but didn't want to change colors without discussion :)

Reply:
Hmmm I think so, I think we would just end up merging them anyway.